python/pyspark/sql/tests.py

```diff
@@ -1886,6 +1887,28 @@ def test_hivecontext(self):
         self.assertTrue(os.path.exists(metastore_path))
 
 
+class SQLTests2(ReusedPySparkTestCase):
```

Comment:
`ReusedPySparkTestCase` launches a class's spark context. So I use it as the default spark context.